nat-lab/tests/utils/ping.py

```diff
@@ -33,18 +34,29 @@ def __init__(self, connection: Connection, ip: str) -> None:
         self._ip = ip
         self._connection = connection
         self._ip_proto = testing.unpack_optional(get_ip_address_type(ip))
+        kill_id = secrets.token_hex(8).upper()
 
         if connection.target_os == TargetOS.Windows:
             self._process = connection.create_process(
                 ["ping", ("-4" if self._ip_proto == IPProto.IPv4 else "-6"), "-t", ip]
```

Comment:
what about windows?

Reply:
Afaics, windows has no support for such thing: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/ping :(